buildenv/jenkins/common/build

```diff
@@ -195,26 +200,50 @@ def checkout_pullrequest() {
             if (omr_upstream) {
                 sh "git config remote.origin.url https://github.com/eclipse/omr.git"
             }
-            checkout_pullrequest(OMR_PR)
+            checkout_pullrequest(OMR_PR, 'eclipse/omr')
         }
     }
 }
 
-def checkout_pullrequest(ID) {
+def checkout_pullrequest(ID, REPO) {
     // Determine if we are checking out a PR or a branch
     try {
         if (ID.isNumber()) {
-            sh "git fetch --progress origin +refs/pull/${ID}/merge:refs/remotes/origin/pr/${ID}/merge"
-            sh "git checkout refs/remotes/origin/pr/${ID}/merge"
+            try {
+                //sh "git fetch --progress origin +refs/pull/${ID}/merge:refs/remotes/origin/pr/${ID}/merge"
```

Comment:
If this is no longer needed, can we remove it?

Reply:
fixed